### DISCUSSION TITLE
Home page totally useless

### DISCUSSION THREAD

**Discussion by ghost** [community/community#110596] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In this account, I'm a corporate developer. My organization has hundreds of private GitHub repos even though I usually only work on 5-7 of them.

I would expect to see on the homepage activities in the repos I usually work in. Especially I want to see a list of open pull requests or dependabot security alerts. I don't want to see random suggestions for other Github projects. As of now, the homepage is totally useless to me.

**Comment by acgandhi** [comment]
+1. The top repositories pane on the left might be useful, if the sorting wasn't completely broken. Why are repos where the last activity (from anyone, including myself) was a year ago being shown above repositories where I pushed earlier today? Why does the recent activity section underneath it just show nothing? The following feed is neat, but does it really need to take 2/3 of the page?

**Reply by roaddemon** [reply]
Seriously.  Top repos seem to be sorted by "date updated closest to one year ago" which is total garbage.  Recent activity is blank despite me actively working in about a dozen repos.  The homepage feed with default filters shows nothing I'm interested in.  Ever.  The whole right panel is gibberish.  The perfect homepage is https://github.com/your_username?tab=repositories

**Comment by aeturnum** [comment]
Totally agree. My home page is currently filled with events from a repo I haven't starred or watched, by people I don't follow, that doesn't tag me. I don't know why it's there and I don't remember ever interacting with the repo.

**Reply by cheezequake** [reply]
If Microsoft owns or controls Github now since the takeover from corporate overlords. This means they are now comfortable with changing things without considering the future. They want financial gain soon so promoting advertised "open-source" projects that will end up benefiting the corporations that very well depend on benefit from freely developed software without paying employees to do it for them, instead just do what Microsoft did when they made Windows 7, steal it all and shut it down in court. 

We will eventually consider not using Github when it all hits the fan and spreads all over the room.

**Reply by cheezequake** [reply]
I am not a corporate developer, but I have seen the path companies have taken when they want something that is hot and popular, they will do what it takes to copy and replicate successful systems aka the "Wall", "Feed", "River" whatever makes it easier to inject more useless crap in front of your face. 

If you KNOW you NEED something, you will find it.
If you DON'T KNOW what you need, you shouldn't be distracted by it. (especially if you have WORK to do!)

Chiselling away. . . on the very ice-berg it floats on.

**Comment by MBurchard** [comment]
Unfortunately, you can't even switch off the stupid suggestions. I have used uBlock and now block the pointless element. It's a shame that you can't do anything useful in this area.
```
github.com##.js-for-you-feed-items
```

**Comment by comedinha** [comment]
Use the old feed script to see yours project activities: https://github.com/Gerrit0/old-github-feed

**Comment by hzxlovec** [comment]
+1， suck design！fuck u

**Comment by roaddemon** [comment]
Yup.  Every element on the homepage is either 100% useless or broken.  Top repositories seems to be sorted by "date modified closest to one year ago", "recent activity" is always blank, and nothing else on the page is relevant to me.  I'd like my landing page to be exactly this https://github.com/your_username?tab=repositories.  Useful and relevant to me.  Although I might regret posting this - github product team might see it and realize they forgot to break that page also.

**Comment by rubengmurray** [comment]
I'd be interested to find out the definition of "Top Repositories". Functionally for me it's not up to scratch.

On the home page what I really want (need) is "Recent Repositories".

This week I've worked on three new organisation repositories and not only do they not show near the top of the "Top Repositories" list for quick access, but they don't show in the list at all when you begin typing.

The primary use case for that section really should be to quickly navigate to the repositories you're mostly likely to want to access quickly.  Repos you've been working on recently are much more likely to fit that description. At the moment is seems confused about it's use case. I have repositories in the quick filter list I last committed to in 2019!

If "Top Repositories" is defined as "Repositories Committed to the Most" then that should be on analytics page somewhere, not your homepage.

**Comment by cybrejon** [comment]
2026 and it's still useless.
